audit/projects/k8s-artifacts-prod/services/enabled.txt

```diff
@@ -9,7 +9,6 @@ cloudtrace.googleapis.com            Cloud Trace API
 compute.googleapis.com               Compute Engine API
 containeranalysis.googleapis.com     Container Analysis API
 containerregistry.googleapis.com     Container Registry API
-containerscanning.googleapis.com     Container Scanning API
```

Comment:
/cc @dims 
was this done manually? I forget where I saw notification that we wanted to disable this now that free pricing is going away

I just re-run `ensure-prod-storage.sh` which still has this service enabled so I suspect it's going to come back

Reply:
@spiffxp i opened an issue to track here : https://github.com/kubernetes/k8s.io/issues/1963 

Dang! :(

cc @ameukam

Reply:
@spiffxp https://github.com/kubernetes/k8s.io/issues/1963. Also check the record meeting of April 28. We had a conservation about this.